pkg/autoupdate/autoupdate.go

```diff
@@ -54,7 +55,7 @@ type Updater struct {
 
 // NewUpdater creates a unstarted updater for a specific binary
 // updated from a TUF mirror.
-func NewUpdater(binaryPath, rootDirectory string, logger log.Logger, opts ...UpdaterOption) (*Updater, error) {
+func NewUpdater(binaryPath, rootDirectory string, gunPrefix string, logger log.Logger, opts ...UpdaterOption) (*Updater, error) {
```

Comment:
This should be an `UpdaterOption`, not a change to the function signature. For example, https://github.com/kolide/launcher/blob/4c0320d61d2b05fdd79bc32e905e2bf670937cfb/pkg/autoupdate/autoupdate.go#L217